prolog/tokenize.pl

```diff
@@ -24,6 +24,7 @@
 text.
 
 */
+:- use_module(library(dcg/basics)).
```

Comment:
let's make the imports here explicit, so a user knows where to look for a rule in use, rather than having to figure it out by process of elimination. (Alternatively, we could use qualified imports, but I think that isn't idiomatic).

Reply:
fixed